package.json

```diff
@@ -1,41 +1,56 @@
 {
-	"name": "metabolica-pathways",
-	"version": "1.0.0",
-	"main": "src/index.js",
-	"dependencies": {
-		"angular": "^1.6.2",
-		"angular-fixed-table-header": "github:daniel-nagy/fixed-table-header",
-		"angular-toastr": "github:Foxandxss/angular-toastr",
-		"d3": "^3.5.17",
-		"escher-vis": "github:nkran/escher",
-		"metabolica": "github:biosustain/metabolica-ui",
-		"jquery": "^3.1.1"
-	},
-	"scripts": {
-		"build": "webpack",
-		"start": "webpack-dev-server"
-	},
-	"devDependencies": {
-		"@types/angular": "^1.6.7",
-		"@types/angular-material": "^1.1.45",
-		"@types/d3": "^4.5.0",
-		"@types/angular-toastr": "^1.6.30",
-		"angular": "^1.6.2",
-		"babel-core": "^6.22.1",
-		"babel-loader": "^6.2.10",
-		"babel-plugin-transform-runtime": "^6.22.0",
-		"babel-preset-es2015": "^6.22.0",
-		"babel-preset-stage-0": "^6.22.0",
-		"css-loader": "^0.26.1",
-		"extract-text-webpack-plugin": "^2.0.0-rc.3",
-		"file-loader": "^0.10.0",
-		"html-loader": "^0.4.4",
-		"html-webpack-plugin": "^2.28.0",
-		"node-sass": "^4.5.0",
-		"sass-loader": "^5.0.1",
-		"ts-loader": "^2.0.0",
-		"typescript": "^2.1.6",
-		"webpack": "^2.2.1",
-		"webpack-dev-server": "^2.3.0"
-	}
+  "name": "metabolica-pathways",
+  "version": "1.0.0",
+  "main": "src/index.js",
+  "dependencies": {
+    "angular": "^1.6.2",
+    "angular-fixed-table-header": "github:daniel-nagy/fixed-table-header",
+    "angular-toastr": "github:Foxandxss/angular-toastr",
+    "d3": "^3.5.17",
+    "escher-vis": "github:DD-DeCaF/escher",
+    "jquery": "^3.1.1",
+    "lodash": "^4.17.4",
+    "metabolica": "github:biosustain/metabolica-ui"
+  },
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack-dev-server",
+    "test": "./node_modules/karma/bin/karma start --browsers=ChromeHeadless",
```

Comment:
I think it would be better to link to `./node_modules/.bin/karma` as NPM creates symlinks to all the binaries there.